stylus/components/table.styl

```diff
@@ -93,23 +93,15 @@ $table-cell--primary
 
 $table-divider
     position sticky
+    z-index 1
```

Comment:
Do we need the `z-index`? And if so shouldn't we use a variable if we do?

Reply:
Not really. It's just to make sure that it goes over relative positionned elements such as the avatar in Contacts. 
Without it any element positionned, whether its relative, absolute, fixed or sticky, and with a `z-index` of 0 that is later in the DOM will go over the divider.

It's not really a layer context like alerts, modals, etc can be.
I guess I could give it the `$alert-index-mobile` which is the lower before 0 but It seems a little bit confusing to me. Or maybe I could add a new `z-index` value of 1 in the settings for cases like this.
What do you think?